.pre-commit-config.yaml

```diff
@@ -0,0 +1,23 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+files: ^tuf/api/
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
```

Comment:
We should consider auto-updating the hooks in this file. Unfortunately, dependabot does not plan to support this (see https://github.com/dependabot/dependabot-core/issues/1524). But there's a [Github Action based workaround](https://browniebroke.com/blog/gh-action-pre-commit-autoupdate/). Or we could use `pre-commit`'s own ci -- [pre-commit.ci](https://pre-commit.ci/). Although I lean towards not adding another CI/CD platform.